woocommerce/woocommerce
# Update/36979

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Added WC Logger to the try/catch block in the "get_order" function in  "class-wc-order-factory.php" 
Closes # .

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:


Was able to recreate the issue by the following steps:

- Manually create a subscription
- Modify the _schedule_start postmeta of the subscription and change it's value into a bool
- This should trigger the error
- Check the log in the WP Admin Dashboard

The admin dashboard should log the exact exception causing the Invalid Subscription error.
